helixbass/tsc-rust
# Variable declaration type checking (part 2)

In this PR:
- Part 2 of type-checking for a variable declaration like `const a: number = true`

To test:
If you run `cargo run tmp.tsx` against a file whose contents are `const a: number = true` you should see a diagnostic debug-logged with `message_text: "Type 'true' is not assignable to type 'number'."`
If you run `cargo run tmp.tsx` against a file whose contents are eg `const a: number = 1` you should see no diagnostics debug-logged
If you follow the testing instructions from #7 you should now see the `message_text: "An arithmetic operand must be of type 'any', 'number', 'bigint' or an enum type."` as part of the debug-logged diagnostic

Based on `generate-other-node-interfaces`
